<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Publications – Symbiosis &amp; Resilience Laboratory</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="./index.html" class="navbar-brand navbar-brand-logo">
    <img src="./Images/University_of_Hong_Kong-Logo.wine_mod4.png" alt="" class="navbar-logo">
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Team.html"> 
<span class="menu-text">Team</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Projects.html"> 
<span class="menu-text">Projects</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./Publications.html" aria-current="page"> 
<span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Resources.html"> 
<span class="menu-text">Resources</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./join-us.html"> 
<span class="menu-text">Join us</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-full page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul class="collapse">
  <li><a href="#section" id="toc-section" class="nav-link active" data-scroll-target="#section">2025</a></li>
  <li><a href="#section-1" id="toc-section-1" class="nav-link" data-scroll-target="#section-1">2024</a></li>
  <li><a href="#section-2" id="toc-section-2" class="nav-link" data-scroll-target="#section-2">2023</a></li>
  <li><a href="#section-3" id="toc-section-3" class="nav-link" data-scroll-target="#section-3">2022</a></li>
  <li><a href="#section-4" id="toc-section-4" class="nav-link" data-scroll-target="#section-4">2021</a></li>
  <li><a href="#section-5" id="toc-section-5" class="nav-link" data-scroll-target="#section-5">2020</a></li>
  <li><a href="#section-6" id="toc-section-6" class="nav-link" data-scroll-target="#section-6">2019</a></li>
  <li><a href="#section-7" id="toc-section-7" class="nav-link" data-scroll-target="#section-7">2018</a></li>
  <li><a href="#section-8" id="toc-section-8" class="nav-link" data-scroll-target="#section-8">2017</a></li>
  <li><a href="#section-9" id="toc-section-9" class="nav-link" data-scroll-target="#section-9">2016</a></li>
  <li><a href="#section-10" id="toc-section-10" class="nav-link" data-scroll-target="#section-10">2015</a></li>
  <li><a href="#section-11" id="toc-section-11" class="nav-link" data-scroll-target="#section-11">2014</a></li>
  <li><a href="#section-12" id="toc-section-12" class="nav-link" data-scroll-target="#section-12">2013</a></li>
  <li><a href="#section-13" id="toc-section-13" class="nav-link" data-scroll-target="#section-13">2012</a></li>
  <li><a href="#section-14" id="toc-section-14" class="nav-link" data-scroll-target="#section-14">2010</a></li>
  <li><a href="#section-15" id="toc-section-15" class="nav-link" data-scroll-target="#section-15">2009</a></li>
  </ul>
</nav>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
    </div>
<!-- main -->
<main class="content column-page-right" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Publications</h1>
</div>



<div class="quarto-title-meta column-page-right">

    
  
    
  </div>
  


</header>


<section id="section" class="level2">
<h2 class="anchored" data-anchor-id="section">2025</h2>
<ol reversed="" start="58">
<li>
<p>Pappalardo P, Hemmi JM, Machida RJ, Leray M, Collins AG, Osborn KJ. 2025. <em>Taxon-specific BLAST percent identity thresholds for identification of unknown sequences using metabarcoding.</em> <strong>Methods in Ecology and Evolution</strong> 16, 2380-2394</p>
</li><li>
<p>O’Dea A, Dillon EM, Brandl SJ, Cramer KL, Cybulski JD, de Gracia B, García-Méndez K, Griswold K, Lin CH, Leray M, Lueders-Dumont JA, Sellers AJ, Wake TA. 2025. <em>Prehistoric archives reveal evidence of predator loss and prey release in Caribbean reef fish communities.</em> <strong>Proceedings of the National Academy of Sciences of the United States of America</strong> 122, e2503986122</p>
</li><li>
<p>Glynn VM, Fernandes de Barros Marangoni L, Guglielmetti M, Tapia ER, Ali V, Quintero H, Rodriguez Guerra EC, Yuval M, Kline DI, Leray M, Connolly SR, Barrett RDH. 2025. <em>The role of holobiont composition and environmental history in thermotolerance of Tropical Eastern Pacific corals.</em> <strong>Current Biology</strong> 35, p3048-3063.e7</p>
</li><li>
<p>Morel-Letelier I, Yuen B, Orellana LH, Kück AC, Camacho-García YE, Lara M, Leray M, Wilkins LGE. 2025. <em>Seasonal transcriptomic shifts reveal metabolic flexibility of chemosynthetic symbionts in an upwelling region.</em> <strong>MSystems</strong> 10 e01686-24</p>
</li></ol>
<hr>
</section>
<section id="section-1" class="level2">
<h2 class="anchored" data-anchor-id="section-1">2024</h2>
<ol reversed="" start="54">
<li>
<p>Rouzé H, Knowlton N, Anker A, Hurt C, Wirshing H, Von Wormhoudt A, Leray M. 2024. <em>An integrated phylogeographic approach for inferring cryptic speciation in the Alpheus lottini species complex, an important coral mutualist.</em> <strong>iScience</strong> 27, 111034</p>
</li><li>
<p>Morel-Letelier I, Yuen B, Kück AC, Camacho-García YE, Petersen JM, Lara M, Leray M, Eisen JA, Osvatic JT, Gros O, Wilkins LGE. 2024. <em>Adaptations to nitrogen availability drive ecological divergence of chemosynthetic symbionts.</em> <strong>PLoS Genetics</strong> 20, e1011295</p>
</li><li>
<p>Seymour M, Guibert I, Jeunen GJ, Helstrom M, Leray M, Elbrecht V, Lau A, How CM, Ho Ip JC, Liu Z, Moffitt J, Qian L, Qin X, Tse CM, Wang Y, Wang Z, Zeng X, Zhao M. 2024. <em>The first International eDNA workshop in Hong Kong: A beginner’s guide for the next generation eDNA researcher.</em> <strong>Environmental DNA</strong> 6, e552</p>
</li><li>
<p>McIlroy SE, Guibert I, Archana A, Haze Chung WY, Duffy JE, Gotama R, Hui J, Knowlton N, Leray M, Meyer CP, Panagiotou G, Paulay G, Russell B, Thompson PD, Baker DM. 2024. <em>Life goes on: spatial heterogeneity promotes biodiversity in an urbanized coastal marine ecosystem.</em> <strong>Global Change Biology</strong> 30, e17248<br>
</p>
</li></ol>
<hr>
</section>
<section id="section-2" class="level2">
<h2 class="anchored" data-anchor-id="section-2">2023</h2>
<ol reversed="" start="50">
<li>
<p>Linsmayer LB, Noel SK, Leray M, Wangpraseurt D, Hassibi C, Kline DI, Tresguerres M. 2023. <em>Effects of bleaching on oxygen dynamics, energy metabolism, and genotype of symbiotic algae in two Caribbean coral species.</em> <strong>Science of The Total Environment</strong> 919, 170753</p>
</li><li>
<p>Weinheimer AR, Aylward FO, Leray M, Scott J. 2023. <em>Contrasting drivers of abundant phage and prokaryotic communities in tropical, coastal ecosystems across the Isthmus of Panama.</em> <strong>ISME Communications</strong> 3, 127</p>
</li><li>
<p>Levy N, Simon-Blecher N, Ben-Ezra S, Yuval M, Doniger T, Leray M, Karako-Lampert S, Tarazi E, Levy O. 2023. <em>Evaluating biodiversity for coral reef reformation and monitoring on complex 3D structures using environmental DNA (eDNA) metabarcoding.</em> <strong>Science of The Total Environment</strong> 856, 159051<br>
</p>
</li></ol>
<hr>
</section>
<section id="section-3" class="level2">
<h2 class="anchored" data-anchor-id="section-3">2022</h2>
<ol reversed="" start="47">
<li>
<p>O’Dea A, Flantua SGA, Leray M, Lueders-Dumont JA, Titcomb MC, 2022. <em>Pleistocene sea level changes and crocodile population histories on the Isthmus of Panama: A comment on Avila-Cervantes et al.</em> <strong>Evolution</strong> 76, 2778–2783</p>
</li><li>
<p>Clever F, Sourisse JM, Preziosi RF, Eisen JA, Rodriguez Guerra EC, Scott JJ, Wilkins LGE, Altieri AH, McMillan WO, Leray M. 2022. <em>The gut microbiome variability of a butterflyfish increases on severely degraded Caribbean reefs.</em> <strong>Communications Biology</strong> 5, 770</p>
</li><li>
<p>Leray M, Knowlton N, Machida R. 2022. <em>MIDORI2: A collection of quality controlled, preformatted, and regularly updated reference databases for taxonomic assignment of eukaryotic mitochondrial sequences.</em> <strong>Environmental DNA</strong> 4, 894–907</p>
</li><li>
<p>Aguilar R, Prakash S, Ogburn MB, Pagenkopp Lohan KM, MacDonald KS III, Driskell AC, Ahyong ST, Leray M, McIlroy SE, Tuckey TD, Baeza JA. 2022. <em>Unresolved taxonomy confounds invasive species identification: the Lysmata vittata Stimpson, 1860 (Decapoda: Caridea: Lysmatidae) species complex and recent introduction of Lysmata vittata sensu stricto in the western Atlantic.</em> <strong>Journal of Crustacean Biology</strong> 42(1), ruab079<br>
</p>
</li></ol>
<hr>
</section>
<section id="section-4" class="level2">
<h2 class="anchored" data-anchor-id="section-4">2021</h2>
<ol reversed="" start="43">
<li>
<p>Gaither MR, DiBattista JD, Leray M, von der Heyden S. 2021. <em>Metabarcoding the marine environment: from single species to biogeographic patterns.</em> <strong>Environmental DNA</strong> 4, 3-8</p>
</li><li>
<p>Leray M, Wilkins LGE, Apprill A, Bik HM, Clever F, Connolly SR, De León ME, Duffy EJ, Ezzat L, Gignoux-Wolfsohn S, Herre EA, Kaye JZ, Kline DI, Kueneman JG, McCormick MK, McMillan OW, O’Dea A, Pereira TJ, Petersen JM, Petticord DJ, Torchin ME, Vega Thurber R, Videvall E, Wcislo WT, Yuen B, Eisen JA. 2021. <em>Natural experiments and long-term monitoring are critical to understand and predict marine host-microbe ecology and evolution.</em> <strong>PLoS Biology</strong> 19, e3001322</p>
</li><li>
<p>Johnson MD, Scott JJ, Leray M, Lucey N, Bravo LMR, Wied WL, Altieri AH. 2021. <em>Rapid ecosystem-scale consequences of acute deoxygenation on a Caribbean coral reef.</em> <strong>Nature Communications</strong> 12, 1240</p>
</li><li>
<p>Osvatic JT, Wilkins LGE, Leibrecht L, Leray M, Zaunera S, Polzina J, Camacho Y, Gros O, van Gils JA, Eisen JA, Petersen JM, Yuen B. 2021. <em>Global biogeography of chemosynthetic symbionts reveals both localized and globally distributed symbiont groups.</em> <strong>Proceedings of the National Academy of Sciences of the United States of America</strong> 118, e2104378118</p>
</li><li>
<p>Wells CD, Paulay G, Nguyen BN, Leray M. 2021. <em>DNA metabarcoding provides insights into the diverse diet of a dominant suspension feeder, the giant plumose anemone Metridium farcimen.</em> <strong>Environmental DNA</strong> 4, 147-156</p>
</li><li>
<p>Rodriguez-Ezpeleta N, Morissette O, Bean C, Manu S, Banerjee P, Lacoursiere A, Ben K, Alter E, Roger F, Holman L, Stewart K, Monaghan M, Mauvisseau Q, Mirimin L, Wangensteen OS, Antognazza C, Helyar S, de Boer H, Monchamp ME, Nijland R, Abbott C, Doi H, Barnes M, Leray M, Hablützel P, Deiner K. 2021. <em>Trade-offs between reducing complex terminology and producing accurate interpretations from environmental DNA: Comment on “Environmental DNA: What’s behind the term?”</em> <strong>Molecular Ecology</strong> 19, 4601-4605</p>
</li><li>
<p>Kline DI, Dehgan A, Bunje P, Selbe S, Chirayath V, Pizarro O, Leray M, Connolly S, Bongaerts P, Treibitz T, Levy O, Kriegman D, Andersson A, McField M, Duffy JE. 2021. <em>The coral reef sentinels program: A Mars shot for blue planet health.</em> <strong>Marine Technology Society Journal</strong> 55, 118-119</p>
</li><li>
<p>Anker AA, Pachelle PPG, Leray M. 2021. <em>Two new species of Typton Costa, 1844 from tropical American waters, with taxonomic notes on T. tortugae McClendon, 1911 and a new record of T. granulosus Ayón-Parente, Hendrickx &amp; Galvan-Villa, 2015 (Decapoda: Caridea: Palaemonidae).</em> <strong>Zootaxa</strong> 4933, 379-390</p>
</li><li>
<p>Anker AA, Leray M, Pachelle PPG. 2021. <em>A new species of Alpheus Fabricius, 1798 from the Caribbean coast of Panama.</em> <strong>Zootaxa</strong> 4933, 379-390<br>
</p>
</li></ol>
<hr>
</section>
<section id="section-5" class="level2">
<h2 class="anchored" data-anchor-id="section-5">2020</h2>
<ol reversed="" start="34">
<li>
<p>Leray M, Knowlton N, Ho S, Nguyen BN, Machida R. 2020. <em>Reply to Locatelli et al.: Evaluating species-level accuracy of GenBank metazoan sequences will require experts’ effort in each group.</em> <strong>Proceedings of the National Academy of Sciences of the United States of America</strong> 117, 32213-32214</p>
</li><li>
<p>Leray M, Machida R. 2020. <em>Seabed mining could come at a high price for a unique fauna.</em> <strong>Molecular Ecology</strong> 29, 4506-4509</p>
</li><li>
<p>Ferreira LAA, Leray M, Anker A. 2020. <em>New findings of the stenopodidean shrimp Microprosthema looensis Goy &amp; Felder, 1988.</em> <strong>Zootaxa</strong> 4729, 445-450</p>
</li><li>
<p>Nguyen BN, Shen EW, Seemann J, Correa AMS, O’Donnell JL, Altieri AH, Knowlton N, Crandall KA, Egan SP, McMillan WO, Leray M. 2020. <em>Environmental DNA survey captures patterns of fish and invertebrate diversity across a tropical seascape.</em> <strong>Scientific Reports</strong> 10:6729<br>
</p>
</li></ol>
<hr>
</section>
<section id="section-6" class="level2">
<h2 class="anchored" data-anchor-id="section-6">2019</h2>
<ol reversed="" start="30">
<li>
<p>Leray M, Knowlton N, Ho SL, Nguyen BN, Machida RJ. 2019. <em>GenBank is a reliable resource for 21st century biodiversity research.</em> <strong>Proceedings of the National Academy of Sciences of the United States of America</strong> 116, 22651-22656</p>
</li><li>
<p>Wilkins LGE, Leray M, Yuen B, Peixoto R, Pereira TJ, Bik HM, Coil DA, Duffy JE, Herre EA, Lessios H, Lucey N, Mejia LC, O’Dea A, Rasher DB, Sharp K, Sogin EM, Thacker RW, Vega Thurber R, Wcislo WT, Wilbanks EG, Eisen JA. 2019. <em>Host-associated microbiomes drive structure and function of marine ecosystems.</em> <strong>PLoS Biology</strong> 17, e3000533</p>
</li><li>
<p>Van Wormhoudt A, Adjeroud M, Rouzé H, Leray M. 2019. <em>Recent and old duplications in crustaceans “Internal Transcribed Spacer 1”: structural and phylogenetic implications.</em> <strong>Molecular Biology Reports</strong> 46, 5185–5195</p>
</li><li>
<p>Leray M, Alldredge AL, Yang JY, Meyer CP, Holbrook SJ, Schmitt RJ, Knowlton N, Brooks AJ. 2019. <em>Dietary partitioning promotes the coexistence of planktivorous species on coral reefs.</em> <strong>Molecular Ecology</strong> 28, 2694–2710</p>
</li><li>
<p>Kandler NM, Wooster MK, Leray M, Knowlton N, de Voogd NJ, Paulay G, Berumen ML. 2019. <em>Hyperdiverse macrofauna communities associated with a common sponge, Stylissa carteri, shift across ecological gradients in the Central Red Sea.</em> <strong>Diversity</strong> 11, 18<br>
</p>
</li></ol>
<hr>
</section>
<section id="section-7" class="level2">
<h2 class="anchored" data-anchor-id="section-7">2018</h2>
<ol reversed="" start="25">
<li>
<p>Galtier d’Auriac I, Quinn RA, Maughan H, Nothias LF, Little M, Kapono CA, Cobian A, Reyes BT, Green K, Quistad SD, Leray M, Smith JE, Dorrestein PC, Rohwer F, Deheyn DD, Hartmann AC. 2018. <em>Before platelets: the production of platelet-activating factor during growth and stress in a basal marine organism.</em> <strong>Proceedings of the Royal Society B: Biological Sciences</strong> 285, 20181307</p>
</li><li>
<p>Pearman JK, Leray M, Villalobos R, Machida R, Berumen M, Knowlton N, Carvalho S. 2018. <em>Cross-shelf investigation of coral reef cryptic benthic organisms reveals diversity patterns of the hidden majority.</em> <strong>Scientific Reports</strong> 8, 8090</p>
</li><li>
<p>Pachelle PPG, Leray M, Anker AA, Lasley R. 2018. <em>Five new records of marine shrimps from the Caribbean coast of Panama.</em> <strong>Zootaxa</strong> 4438, 128-136</p>
</li><li>
<p>Leray M, Ho S-L, Lin J, Machida R. 2018. <em>MIDORI server: a webserver for taxonomic assignment of unknown metazoan mitochondrial-encoded sequences using a curated database.</em> <strong>Bioinformatics</strong> 34, bty454<br>
</p>
</li></ol>
<hr>
</section>
<section id="section-8" class="level2">
<h2 class="anchored" data-anchor-id="section-8">2017</h2>
<ol reversed="" start="21">
<li>
<p>Ransome E, Geller JB, Timmers M, Leray M, Mahardini A, Sembiring A, Collins AG, Meyer CP. 2017. <em>The importance of standardization for biodiversity comparisons, A case study using autonomous reef monitoring structures (ARMS) and metabarcoding to measure cryptic diversity on Mo’orea coral reefs, French Polynesia.</em> <strong>PLoS One</strong> 12,e0175066</p>
</li><li>
<p>Rouzé H, Leray M, Magalon H, Penin L, Gélin P, Knowlton N, Fauvelot C. 2017. <em>Molecular characterization reveals the complexity of previously overlooked coral-exosymbiont interactions and the implications for coral-guild ecology.</em> <strong>Scientific Report</strong> 7,44923</p>
</li><li>
<p>Leray M, Knowlton N. 2017. <em>Random sampling causes the low reproducibility of rare eukaryotic OTUs in Illumina COI metabarcoding.</em> <strong>PeerJ</strong> 5,e3006</p>
</li><li>
<p>Machida RJ, Leray M, Ho S-L, Knowlton N. 2017. <em>Metazoan mitochondrial gene sequence reference datasets for taxonomic assignment of environmental samples.</em> <strong>Scientific Data</strong> 4,170027</p>
</li></ol>
<hr>
</section>
<section id="section-9" class="level2">
<h2 class="anchored" data-anchor-id="section-9">2016</h2>
<ol reversed="" start="17">
<li>
<p>Leray M, Knowlton N. 2016. <em>Censusing marine eukaryotic diversity in the twenty-first century.</em> <strong>Philosophical Transactions of the Royal Society B</strong> 371,20150331</p>
</li><li>
<p>Al-Rshaidat MMD, Snider A, Rosebraugh S, Devine AM, Devine TD, Plaisance L, Knowlton N, Leray M. 2016. <em>Deep COI sequencing of standardized benthic samples unveils overlooked diversity of Jordanian coral reefs in the northern Red Sea.</em> <strong>Genome</strong> 59,724-737</p>
</li><li>
<p>Leray M, Knowlton N. 2016. <em>Visualizing patterns of marine eukaryotic diversity from metabarcoding data using QIIME.</em> <strong>In, Bourlat S.J. (eds) Methods in Molecular Biology, Marine Genomics Methods and Protocols. Springer Protocols</strong>, pp 219-235</p>
</li><li>
<p>Bourlat SJ, Haenel Q, Finnman J, Leray M. 2016. <em>Preparation of amplicon libraries for metabarcoding of marine eukaryotes using Illumina MiSeq, The dual-PCR method.</em> <strong>In, Bourlat S.J. (eds) Methods in Molecular Biology, Marine Genomics Methods and Protocols. Springer Protocols</strong>, pp 197-207</p>
</li><li>
<p>Leray M, Haenel Q, Bourlat SJ. 2016. <em>Preparation of amplicon libraries for metabarcoding of marine eukaryotes using Illumina MiSeq, The adapter ligation method.</em> <strong>In, Bourlat S.J. (eds) Methods in Molecular Biology, Marine Genomics Methods and Protocols. Springer Protocols</strong>, pp 209-218<br>
</p>
</li></ol>
<hr>
</section>
<section id="section-10" class="level2">
<h2 class="anchored" data-anchor-id="section-10">2015</h2>
<ol reversed="" start="12">
<li>
<p>Leray M, Meyer CP, Mills SC. 2015. <em>Metabarcoding dietary analysis of coral dwelling predatory fish demonstrates the minor contribution of coral mutualists to their highly partitioned, generalist diet.</em> <strong>PeerJ</strong> 3, e1047</p>
</li><li>
<p>Knowlton N, Leray M. 2015. <em>Exploring coral reefs using the tools of molecular genetics.</em> <strong>In, Birkeland C. (eds) Coral Reefs in the Anthropocene. Springer Science+Business Media, Dordrecht</strong>, pp 117-132</p>
</li><li>
<p>Leray M, Knowlton N. 2015. <em>DNA barcoding and metabarcoding of standardized samples reveal patterns of marine benthic diversity.</em> <strong>Proceedings of the National Academy of Sciences of the United States of America</strong> 112, 2076-2081<br>
</p>
</li></ol>
<hr>
</section>
<section id="section-11" class="level2">
<h2 class="anchored" data-anchor-id="section-11">2014</h2>
<ol reversed="" start="9">
<li>
<p>Stier AC, Leray M. 2014. <em>Predators alter community organization of coral reef cryptofauna and reduce abundance of coral mutualists.</em> <strong>Coral Reefs</strong> 33,181-191<br>
</p>
</li></ol>
<hr>
</section>
<section id="section-12" class="level2">
<h2 class="anchored" data-anchor-id="section-12">2013</h2>
<ol reversed="" start="8">
<li>
<p>Leray M, Yang JY, Meyer CP, Mills SC, Agudelo N, Ranwez V, Boehm JT, Machida RJ. 2013. <em>A new versatile primer set targeting a short fragment of the mitochondrial COI region for metabarcoding metazoan diversity, application for characterizing coral reef fish gut contents.</em> <strong>Frontiers in Zoology</strong> 10,34</p>
</li><li>
<p>Leray M, Agudelo N, Mills SC, Meyer CP. 2013. <em>Effectiveness of annealing blocking primers versus restriction enzymes for characterization of generalist diets, unexpected prey revealed in the gut contents of two coral reef fish species.</em> <strong>PLoS One</strong> 8,e58076<br>
</p>
</li></ol>
<hr>
</section>
<section id="section-13" class="level2">
<h2 class="anchored" data-anchor-id="section-13">2012</h2>
<ol reversed="" start="6">
<li>
<p>Leray M, Boehm JT, Mills SC, Meyer CP. 2012. <em>Moorea BIOCODE barcode library as a tool for understanding predator-prey interactions, insights into the diet of common predatory coral reef fishes.</em> <strong>Coral Reefs</strong> 31,383-388</p>
</li><li>
<p>Leray M, Beraud M, Anker A, Chancerelle Y, Mills S. 2012. <em>Acanthaster planci outbreak, decline in coral health, coral size structure modification and consequences for obligate decapod assemblages.</em> <strong>PLoS One</strong> 7,e35456</p>
</li><li>
<p>Stier AC, Gil MA, McKeon CS, Lemer S, Leray M, Mills SC, Osenberg CW. 2012. <em>Housekeeping mutualisms, do more symbionts facilitate host performance?</em> <strong>PLoS One</strong> 7,e32079<br>
</p>
</li></ol>
<hr>
</section>
<section id="section-14" class="level2">
<h2 class="anchored" data-anchor-id="section-14">2010</h2>
<ol reversed="" start="3">
<li>
<p>Leray M, Beldade R, Holbrook SJ, Schmitt RJ, Planes S, Bernardi G. 2010. <em>Allopatric divergence and speciation in coral reef fish, the three-spot Dascyllus, Dascyllus trimaculatus, species complex.</em> <strong>Evolution</strong> 64,1218-1230</p>
</li><li>
<p>Van Herwerden L, Howard Choat J, Newman SJ, Leray M, Hillersøy G. 2009. <em>Complex patterns of population structure and recruitment of Plectropomus leopardus (Pisces, Epinephelidae) in the Indo-West Pacific, Implications for fisheries management.</em> <strong>Marine Biology</strong> 156,1595-1607</p>
</li></ol>
<hr>
</section>
<section id="section-15" class="level2">
<h2 class="anchored" data-anchor-id="section-15">2009</h2>
<ol start="1">
<li>
<p>Leray M, Beldade R, Holbrook SJ, Schmitt RJ, Planes S, Bernardi G. 2009. <em>Isolation and characterization of 13 polymorphic nuclear microsatellite primers for the widespread Indo‐Pacific three‐spot damselfish, Dascyllus trimaculatus, and closely related D. auripinnis.</em> <strong>Molecular ecology resources</strong> 9, 213-215<br>
</p>
</li></ol>
<hr>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>